Inspicio/Models/Image.cs

```diff
@@ -9,15 +10,25 @@ namespace Inspicio.Models
     public class Image
     {
         public int ImageID { get; set; }
+
         public string OwnerId { get; set; }
+
         public string Content { get; set; }
+
+        [DisplayName("Dislikes")]
         public int DownRating { get; set; }
```

Comment:
Might be nice to display an image instead of display name e.g thumbs up/thumbs down. For MVP, this won't be necessary.